lib/dlss/capistrano/tasks/sidekiq.rake

```diff
@@ -0,0 +1,51 @@
+# Capistrano plugin hook to set default values
+namespace :load do
+  task :defaults do
+    set :sidekiq_role, fetch(:sidekiq_role, :app)
+  end
+end
+
+# Integrate sidekiq hooks into Capistrano
+namespace :deploy do
+  before :starting, :add_sidekiq_hooks do
+    invoke 'sidekiq:add_hooks'
+  end
+end
+
+namespace :sidekiq do
```

Comment:
Since this is specific to how we deploy our systems (systemd approach only) is there a way we could name this something to indicate that? My reasoning here is 6 months from now me on an unmigrated system might just not remember which and how this should work.

Reply:
Yup. That's one of the suggestions @cbeer originally made, too. I spaced on how sul_requests, content_search, and exhibits were set up to manage sidekiq. I'll make this change, no problem.

Reply:
done-zo now, thanks